torch/_torch_docs.py

```diff
@@ -1987,15 +1987,18 @@ def parse_kwargs(desc):
            r"""
 inverse(input, out=None) -> Tensor
 
-Takes the inverse of the square matrix :attr:`input`.
+Takes the inverse of the square matrix :attr:`input`. :attr:`input` can be batches
+of 2D square tensors, in which case this function would return a tensor composed of
+individual inverses.
```

Comment:
What happens if the inverse can't be computed? Just wondering

Reply:
An error will be thrown. We have checks in internal code to take care of this.